Modules/Loadable/Markups/VTKWidgets/vtkSlicerMarkupsWidgetRepresentation2D.cxx

```diff
@@ -157,6 +175,24 @@ void vtkSlicerMarkupsWidgetRepresentation2D::GetSliceToWorldCoordinates(const do
   worldPos[2] = rasw[2]/rasw[3];
 }
 
+//----------------------------------------------------------------------
+void vtkSlicerMarkupsWidgetRepresentation2D::GetBackgroundVolumeXYCenter(double centerPos[2])
```

Comment:
We cannot assume that there is a volume in the scene (the leading lines should be correctly and consistently displayed if there is no volumes or many volumes in the scene). Instead of using the volume center as center of projection point, we could use the center of the bounding box of the markup's control points.